scripts/tasks/0_init_config.ipynb

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    network_share = Path(r"")  # e.g. r"L:/" or r"/mnt/data", OS-specific.
</pre><p>This is very system specific, and depending on the laptop (especially on macOS) it might mount the share in a different place. I think this is more generic, anyone can then come and paste the path to the mounted network share.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mscheltienne/eeg-flow/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>